### DISCUSSION TITLE
Ignore specific IP addresses for clicks & stats

### DISCUSSION THREAD

**Discussion by matbe81** [YOURLS/YOURLS#3550] [Q&A] [answered]
Is there a way to ignore given ip addresses in counting clicks + stats to avoid local tests being counted?
I know the public IP addresses

Thank you.

**Comment by dgw** [comment, marked as answer]
If your local tests occur while you are logged in to YOURLS, https://github.com/dgw/yourls-dont-track-admins would help.

Otherwise, that plugin can serve as a starting point, to be modified so it checks the requesting IP instead of the authentication status.

**Reply by matbe81** [reply]
Totally acceptable solution. Thank you.
